src/js/tools/metroconfig.ts

```diff
@@ -0,0 +1,119 @@
+import type { MetroConfig, MixedOutput, Module, ReadOnlyGraph } from 'metro';
```

Comment:
Hey, I haven't found any tests covering rn.patch.metro.config.js, this file already exists but it should be covered by tests if it's going to be executed on the user side. What do you think? (It could be done on another issue/PR)

Reply:
All the rn.patch* patch files are only for the e2e test that why there are no tests for the patches themselves.